### ISSUE TITLE
v2.1.5 RE2 Claire A (PC as Ada) room 201 crashed

### ISSUE THREAD

**Issue by otamega72** [IntelOrca/biorand#162]
BioRand 2.1.5
    Seed: R522-ZCZ2-ZTZMP7ZLL703
    Player: 1 Claire
    Scenario: A
  
Creating area from 60A to 300:
        Connected 60A:1 (unblock) to 201:0 **<---OK**
        Connected 201:1 to 11A:1  **<-----CRASH**

201 (0xD8E) opcode removed
    201 (0xD6A) opcode removed
    201 (0xD86) opcode removed
    201 (0xD8A) opcode removed
    201 (0xD8E) opcode removed
    201 (0xDA4) opcode removed
    201 (0xDA7) opcode removed
    201 (0xDA8) opcode removed
    201 (0xDAC) opcode removed
    201 (0xDB0) opcode removed
    201 (0xDB2) opcode removed
    201 (0xDB4) opcode removed
    Created 201:1 (0x0) [GiantMoth,0,0,23,0] at (-20363,0,-24479) 

CRASHED here
![Capture](https://user-images.githubusercontent.com/120957601/212519863-f15a1dba-f63a-4d7a-88e5-2bd28ff6859b.JPG)

**Comment by IntelOrca** [comment]
Duplicate of #153
